apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,326 @@
+                            }
+                            type="text"
+                            maxLength={
+                                IdentityProviderManagementConstants
+                                    .AUTHENTICATOR_SETTINGS_FORM_FIELD_CONSTRAINTS.CLIENT_SECRET_MAX_LENGTH as number
+                            }
+                            minLength={
+                                IdentityProviderManagementConstants
+                                    .AUTHENTICATOR_SETTINGS_FORM_FIELD_CONSTRAINTS.CLIENT_SECRET_MIN_LENGTH as number
+                            }
+                            data-testid={ `${ testId }-idp-base-url` }
+                            width={ 13 }
+                        />
+                        <Field.Input
+                            ariaLabel="HYPR API Token"
+                            inputType="password"
+                            className="addon-field-wrapper"
+                            name="apiToken"
+                            label={
+                                t("console:develop.features.authenticationProvider.forms" +
+                                    ".authenticatorSettings.hypr.apiToken.label")
+                            }
+                            placeholder={
+                                t("console:develop.features.authenticationProvider.forms" +
+                                    ".authenticatorSettings.hypr.apiToken.placeholder")
+                            }
+                            required={ true }
+                            message={
+                                t("console:develop.features.authenticationProvider.forms" +
+                                    ".authenticatorSettings.hypr.apiToken.validations.required")
+                            }
+                            type="password"
+                            hidePassword={ t("common:hide") }
+                            showPassword={ t("common:show") }
+                            maxLength={
+                                IdentityProviderManagementConstants
+                                    .AUTHENTICATOR_SETTINGS_FORM_FIELD_CONSTRAINTS.CLIENT_SECRET_MAX_LENGTH as number
+                            }
+                            minLength={
+                                IdentityProviderManagementConstants
+                                    .AUTHENTICATOR_SETTINGS_FORM_FIELD_CONSTRAINTS.CLIENT_SECRET_MIN_LENGTH as number
+                            }
+                            data-testid={ `${ testId }-idp-api-token` }
+                            width={ 13 }
+                        />
+                    </WizardPage>
+                </Wizard>
+            )
+            : null
+    );
+};
+
+/**
+ * Default props for the Microsoft Authentication Provider Create Wizard Page Component.
+ */
+HyprAuthenticationProviderCreateWizardContent.defaultProps = {
+    "data-testid": "microsoft-idp-create-wizard-page"
```

Comment:
Check if we can use data-componentid

Reply:
addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)